example/ecr-copy/main.go

```diff
@@ -17,7 +17,7 @@ package main
 
 import (
 	"context"
-	"os"
+	"flag"
```

Comment:
:frowning_face: I really dislike the "flag" library.

What about an environment variable instead?  I'm already using one in ecr-pull to control parallelism anyway.

Reply:
I know you enjoy its `-flags` and its API :paintbrush: 

I'll update to use environment variables. I only used `flag` because "`stdlib`" is close at hand here.